samples/protobuf-sample/README.md

```diff
@@ -1,3 +1 @@
-# VLC sample
-
-code taken for regression testing from https://github.com/protocolbuffers/protobuf
+Code under src/ taken from https://github.com/protocolbuffers/protobuf. Rights except redistribution reserved to Google. See LICENSE.
```

Comment:
"Rights except redistribution reserved to Google." is incorrect.  The license is BSD-3-Clause

Reply:
Okay, I'll go ahead and correct every instance.